api/v1alpha1/networkchaos_types.go

```diff
@@ -266,17 +270,17 @@ type DelaySpec struct {
 }
 
 // ToNetem implements Netem interface.
-func (delay *DelaySpec) ToNetem() (*chaosdaemonpb.Netem, error) {
-	delayTime, err := time.ParseDuration(delay.Latency)
+func (in *DelaySpec) ToNetem() (*chaosdaemonpb.Netem, error) {
```

Comment:
Why change all to `in`? `in` is not that readable...

Reply:
I think it's because IDE like goland would show warning for having all method receivers using same name. `in` is used by generator when generating `DeepCopy`

Reply:
Yes,this is caused by goland and generator.